mac_messages_mcp/messages.py

```
@@ -251,22 +251,35 @@ def _looks_like_phone_input(value: str) -> bool:
-    "\U0001f780-\U0001f7ff"  # Geometric Shapes
-    "\U0001f800-\U0001f8ff"  # Supplemental Arrows-C
-    "\U0001f900-\U0001f9ff"  # Supplemental Symbols and Pictographs
-    "\U0001fa00-\U0001fa6f"  # Chess Symbols
-    "\U0001fa70-\U0001faff"  # Symbols and Pictographs Extended-A
-    "\U00002702-\U000027b0"  # Dingbats
-    "\U000024c2-\U0001f251"
-    "]+"
+# Inclusive (start, end) code-point ranges for characters stripped as emoji.
+# Kept as ordinal comparisons so CodeQL py/overly-large-range does not treat
+# supplementary-plane regex ranges as overlapping U+FFFD, and so the old
+# catch-all U+24C2–U+1F251 span cannot swallow CJK and other non-emoji text.
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)